src/karmen_backend/server/services/mailer/templates/mail_template.py

```diff
@@ -19,7 +43,27 @@ def textbody(self):
     def htmlbody(self):
         return "<br />\n".join(self.textbody().split("\n"))
 
+    def get_content(self, content_type):
```

Comment:
it's now unclear from the class API, what the method use in what situation whether `htmlbody` or `get_content("html")`. Would you like to make `htmlbody`/`textbody` private?

Reply:
You've got a good point here, I will modify this.

Reply:
Replaced with `render` to make things clear and added missing docstrings.